selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/intelligent/CommandsEditor.java

```diff
@@ -423,33 +415,36 @@ public String getAllVisibleTextFromMacrosContainer() {
 
   public void enterMacroCommandByEnter(String item) {
     selectMacroCommand(item);
-    actionsFactory.createAction(seleniumWebDriver).sendKeys(Keys.ENTER.toString()).perform();
+    seleniumWebDriverHelper.sendKeys(ENTER.toString());
   }
 
   public void enterMacroCommandByDoubleClick(String item) {
     selectMacroCommand(item);
-    actionsFactory.createAction(seleniumWebDriver).doubleClick().perform();
+    seleniumWebDriverHelper.doubleClick();
   }
 
   public void selectMacroCommand(String item) {
-    String locator =
-        String.format(CommandsLocators.COMMAND_MACROS_FORM + "//div[text()='%s']", item);
-    redrawWait.until(visibilityOfElementLocated(By.xpath(locator))).click();
-    WebElement highlightEItem = seleniumWebDriver.findElement(By.xpath(locator));
-    redrawWait
-        .until(visibilityOf(highlightEItem))
-        .getCssValue("background-color")
-        .contains("rgb(37, 108, 159)");
-  }
-
-  public void waitMacroCommandIsSelected(String item) {
-    String locator =
-        String.format(CommandsLocators.COMMAND_MACROS_FORM + "//div[text()='%s']", item);
-    WebElement highlightEItem = seleniumWebDriver.findElement(By.xpath(locator));
-    redrawWait
-        .until(visibilityOf(highlightEItem))
-        .getCssValue("background-color")
-        .contains("rgb(37, 108, 159)");
+    waitVisibilityAndGetMacrosCommand(item).click();
+
+    waitMacrosCommandIsSelected(item);
+  }
+
+  private void waitMacrosCommandIsSelected(String item) {
+    webDriverWaitFactory
+        .get()
+        .until(
+            (ExpectedCondition<Boolean>)
+                driver -> {
+                  String backgroundColor =
+                      waitVisibilityAndGetMacrosCommand(item).getCssValue("background-color");
+                  return backgroundColor.contains("rgba(215, 215, 215, 0.2)");
+                });
+  }
+
+  public WebElement waitVisibilityAndGetMacrosCommand(String item) {
```

Comment:
`MacrosCommand` sounds misleading.
Let's name it "CommandMacro" or just "Macro" due to [Eclipse Che documentation](https://www.eclipse.org/che/docs/ide/commands/index.html#macros): `waitVisibilityAndGetMacro()`